cli/pkg/cicd/updater/vmclarity.go

```diff
@@ -0,0 +1,96 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package updater
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/runtime_scan/pkg/utils"
+	"github.com/openclarity/vmclarity/shared/pkg/backendclient"
+)
+
+type VMClarityUpdater struct {
+	client       *backendclient.BackendClient
+	scanID       string
+	scanResultID string
+}
+
+func NewVMClarityUpdater(
+	client *backendclient.BackendClient, scanID, scanResultID string) (*VMClarityUpdater, error) {
+	if client == nil {
+		return nil, errors.New("backend client must not be nil")
+	}
+	return &VMClarityUpdater{
+		client:       client,
+		scanID:       scanID,
+		scanResultID: scanResultID,
+	}, nil
+}
+
+func (u *VMClarityUpdater) UpdateScan(ctx context.Context) error {
+	scan, err := u.updatedScanSummary(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to update scan results summary: %v", err)
+	}
+
+	scan.EndTime = utils.PointerTo(time.Now())
+	scan.State = utils.PointerTo(models.ScanStateDone)
+	scan.StateMessage = utils.PointerTo("All scan jobs completed")
```

Comment:
`All scan jobs completed` - we need to make it const that will be used in all places.